tests/test_hierarchy_device.cu

```diff
@@ -426,7 +426,7 @@ BOOST_AUTO_TEST_CASE(amgx)
     // We only supports Jacobi smoother on the device
     params->put("smoother.type", "Jacobi");
 
-    double const tolerance = 1.;
+    double const tolerance = 5.;
```

Comment:
Please call it tolerance_percent.  I jumped when I saw it initially when I thought it might be relative tolerance

Reply:
Sure